runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -329,6 +329,18 @@ protected void startUp()
 
 				invokeOnMainThread(() ->
 				{
+					// Get and display the device and driver used by the GPU plugin
+					GLDrawable dummyDrawable = GLDrawableFactory.getFactory(GLProfile.getDefault())
+						.createDummyDrawable(GLProfile.getDefaultDevice(), true, new GLCapabilities(GLProfile.getDefault()), null);
+					dummyDrawable.setRealized(true);
+					GLContext versionContext = dummyDrawable.createContext(null);
+					versionContext.makeCurrent();
+					GL versionGL = versionContext.getGL().getGL();
```

Comment:
`.getGL().getGL()`?

Reply:
Yup, it's incredible.

If you just do a single `.getGL()`, the code fails with an NPE when using `versionGL`, so it looks like it returns a `null`, but if you call it twice, it works entirely as intended. This is likely some JOGL spaghetti; I'll go add a comment.

Reply:
it's not possible for `getGL()` to return null but `getGL().getGL()` to not throw a npe.